pylint.sh

```diff
@@ -1,16 +0,0 @@
-#!/bin/sh
```

Comment:
This file might be still useful if you want to run pylint (static checker) for whole project.